src/telemetry/metrics/labels/mod.rs

```diff
@@ -54,7 +54,7 @@ pub struct RequestLabels {
     direction: Direction,
 
     // Additional labels identifying the destination service of an outbound
-    // request, provided by the Conduit control plane's service discovery.
+    // request, provided by the control plane's service discovery.
```

Comment:
I would remove "the control plane's" here because whether or not there's a control plane is irrelevant in this module.